src/components/MiddleEllipsis/MiddleEllipsis.tsx

```diff
@@ -1,18 +1,18 @@
 import { Box, BoxProps } from '@chakra-ui/react'
-import React from 'react'
-import ReactMiddleEllipsis from 'react-middle-ellipsis'
 
 type MiddleEllipsisProps = {
-  children: React.ReactNode
-} & ({ maxWidth: string } | { width: string }) &
-  BoxProps
+  address: string
+  shouldShorten?: boolean
+} & BoxProps
 
-export const MiddleEllipsis = ({ children, maxWidth, width, ...rest }: MiddleEllipsisProps) => {
+export const MiddleEllipsis = ({ address, shouldShorten = true, ...rest }: MiddleEllipsisProps) => {
   return (
-    <Box whiteSpace='nowrap' maxWidth={maxWidth} width={width} {...rest}>
-      <ReactMiddleEllipsis>
-        <span>{children}</span>
-      </ReactMiddleEllipsis>
+    <Box whiteSpace='nowrap' {...rest}>
+      <span>{shouldShorten ? shortenAddress(address) : address}</span>
     </Box>
   )
 }
+
+export function shortenAddress(address: string): string {
```

Comment:
if we're doing it this way...

let's use the existing version https://github.com/shapeshift/web/blob/develop/src/state/slices/portfolioSlice/utils.ts#L50

Reply:
done